src/features/change-password/ui/ChangePasswordForm.tsx

```diff
@@ -32,7 +31,7 @@ const ChangePasswordForm: FC<Props> = props => {
   } = useChangePasswordMutation({
     onSuccess: () => {
       props.onChangePasswordSuccess();
-      toast.show(t('auth:password_updated'));
+      Toast.show({ type: 'toast', text1: t('auth:password_updated') });
```

Comment:
Not sure if this should be a toast or a banner but we should probably use one of those hooks here

```tsx
banner.show(t('auth:password_updated'), { type: 'toast' });
```